packages/@azure/servicebus/data-plane/lib/core/messageReceiver.ts

```diff
@@ -81,10 +81,11 @@ export interface ReceiveOptions extends MessageHandlerOptions {
    */
   receiveMode?: ReceiveMode;
   /**
-   * @property {number} [maxConcurrentCalls] The maximum number of messages that should be
-   * processed concurrently while in peek lock mode. Once this limit has been reached, more
-   * messages will not be received until messages currently being processed have been settled.
-   * - **Default**: `1` (message at a time).
+   * @property {number} [maxConcurrentCalls] The maximum number of messages that can be
```

Comment:
This property is not yet exposed to the user.
To do so, we need to move it out of `ReceiveOptions` and into `MessageHandlerOptions`

Reply:
got it, fixed